documentation/navitia.io/source/integration.rst

```diff
@@ -287,21 +289,24 @@ Collections
 Specific parameters
 ###################
 
-There are som specific parameters.
+There are other specific parameters.
 A least, there is one: *odt_level* which can be applied only on /lines collection...
 
-It allows you to request navitia for specific pick up lines. "odt_level" can take one of these values:
+It allows you to request navitia for specific pick up lines. It refers to the `odt`_ section.
```

Comment:
whaou we can query navitia for that ? 

`api.navitia.io/pickup_lines`

```
[{ 
line:"Is your dad a terrorist? Because you are the bomb.", 
efficiency: 0.001%
}]
```